http/miscellaneous/balada-injector-malware.yaml

```
@@ -0,0 +1,32 @@
+id: balada-injector-malware
+
+info:
+  name: Balada Injector Malware - Detect
+  author: kazet
+  severity: high
+  description: |
+    Checks websites for Balada Injector malware.
+  reference:
+    - https://blog.sucuri.net/2024/01/thousands-of-sites-with-popup-builder-compromised-by-balada-injector.html
+  metadata:
+    max-request: 1
+  tags: malware,balada,misc
+
+http:
+  - method: GET
+    path:
+      - "{{BaseURL}}"
+
+    redirects: true
+
+    matchers-condition: and
+    matchers:
+      - type: regex
+        part: body
+        regex:
+          - '(?mi)sgpbWillOpen", function\(e\) {if \(e.detail.popupId.{0,100}eval.{0,100}atob'
```

```suggestion
          - '(?mi)sgpbWillOpen", *function\(e\) *{if *\(e[.]detail[.]popupId.{0,100}eval.{0,100}atob'
```
I guess (definitely make spaces optional or it just won't match)